pyflagser/tests/conftest.py

```diff
@@ -5,6 +5,7 @@
 from ..modules.flagser_pybind import AVAILABLE_FILTRATIONS
 
 files_with_filtration_results = ["d5.flag"]
+extensive_test = ['medium-test-data.flag', 'd10.flag']
```

Comment:
Shall we remove this line?

Reply:
We can, I let it because if in the future we would like to have 2 types of test suite (quick, extensive), we already know which dataset are the extensive ones. But we can remove it.

Reply:
I'd suggest tp remove it not to confuse anyone who hasn't been following the discussions in this thread, but it's not important.